apps/sms/views/conversation/js/conversation.js

```diff
@@ -688,7 +688,7 @@ var ConversationView = {
       // Populate draft if there is one
       // TODO merge with handleDraft ? Bug 1164431
       Drafts.request().then(() => {
-        this.draft = Threads.get(threadId).getDraft();
+        this.draft = Threads.get(threadId) && Threads.get(threadId).getDraft();
```

Comment:
I think it's better to figure out real reason ;) We can do it later if we have the same issue in master